content/sensu-core/2.0/reference/assets.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Assets"
+description: "The assets reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Assets
+
+## What are Assets?
+
+Sensu Assets are resources that checks can specify as dependencies. When an
+Agent runs a Check, it will ensure that all of the Check's required Assets
+are available to the Agent. If they aren't, the Agent will install them by
+consulting each of the Assets' URLs.
+
+## Asset management
+
+Assets are created through the `sensuctl` CLI.
```

Comment:
Do they have an API endpoint you can mention as well?

Reply:
Yes they do, as do all 2.x resources. I wasn't sure if these docs were meant to serve as API docs. I think someone told me to avoid doing that here, but I may have dreamed it. :)